### PR TITLE
October 2020 Release Notes for Azure Core and Search

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: releases/2020-10/java.md [hunk 1]
```diff
@@ -36,9 +94,57 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+### Azure Core Test ([CHANGELOG](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-test/CHANGELOG.md#150-2020-10-01))
+
+#### New Features
+
+- Enhanced playback recording to use test class name plus test name to identify records.
+
+#### Key Bug Fixes
+
+- Added additional response data redaction.
+- Updated handling of `HttpClient` retrieval from the classpath to no longer require dependent libraries to `add-opens` in Java 9+.
```
Comment (samvaity) [inline releases/2020-10/java.md:126]: Can these be skipped?

File: releases/2020-10/java.md [hunk 2]
```diff
@@ -36,9 +94,57 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+##### Breaking Changes
 
-- Major changes only!
+- Renamed `SearchBatchClient` and `SearchBatchAsyncClient` to `SearchIndexingBufferedSender` and `SearchIndexingBufferedAsyncSender`.
+- Removed `SearchBatchClientBuilder` for options bag `SearchIndexingBufferedSenderOptions`.
+- Renamed `getSearchBatchClient` to `getSearchIndexingBufferedSender` in `SearchClient`.
+- Made `SearchIdexingBufferedSender` generic typed.
+- Removed `IndexingHooks` in favor of individual callbacks.
+- Removed the ability to configure `batchSize` on buffered sender and changed the default to 500 instead of 1000.
+- Changed `onActionRemoved` to `onActionSent`.
```
Comment (samvaity) [inline releases/2020-10/java.md:147]: Renamed?

File: releases/2020-10/java.md [hunk 3]
```diff
@@ -36,9 +94,57 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- Added `ClientOptions` to all builders to support setting `applicationId` in `User-Agent` string and headers that need to be applied to each request.
+- Added support for `HttpPipelinePosition` in client builders to determine when an `HttpPipelinePolicy` will be invoked.
+
+##### Breaking Changes
 
-- Major changes only!
+- Renamed `SearchBatchClient` and `SearchBatchAsyncClient` to `SearchIndexingBufferedSender` and `SearchIndexingBufferedAsyncSender`.
+- Removed `SearchBatchClientBuilder` for options bag `SearchIndexingBufferedSenderOptions`.
+- Renamed `getSearchBatchClient` to `getSearchIndexingBufferedSender` in `SearchClient`.
+- Made `SearchIdexingBufferedSender` generic typed.
```
Comment (samvaity) [inline releases/2020-10/java.md:144]: Should this be noteworthy for release notes?

Comment (alzimmermsft) [reply releases/2020-10/java.md:144]: General functionality was released in a previous beta, so it may not be worth mentioning again.

Comment (samvaity) [reply releases/2020-10/java.md:139]: Should we remove this section?

**Comment by samvaity** [comment]
Closing this in favor of https://github.com/Azure/azure-sdk/pull/1915

**Pull request closed without merging**